cmake/scripts/linux/ArchSetup.cmake

```diff
@@ -44,3 +44,7 @@ endif()
 if(ENABLE_MIR)
   set(ENABLE_VDPAU OFF CACHE BOOL "Disabling VDPAU since no Mir support" FORCE)
 endif()
+
+if(CPU MATCHES 64)
+  set(ENABLE_JEMALLOC ON CACHE BOOL "Enable jmalloc allocator for 64-bit linux builds" FORCE)
```

Comment:
that didn't work for me, so I assumed you wanted it forced-enabled on aarch64/x86_64.